django_loose_fk/inspectors/fields.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+from drf_yasg import openapi
+from drf_yasg.inspectors.base import NotHandled
+from drf_yasg.inspectors.field import FieldInspector
+
+from django_loose_fk.drf import FKOrURLField
+
+logger = logging.getLogger(__name__)
+
+
+TYPES_MAP = {
```

Comment:
this is not used anywhere, right?

Reply:
yeah, you are right, It's copypaste thing